selenium_tests/CanonizerBrowsePage.py

```diff
@@ -436,15 +440,78 @@ def select_by_value_government_sandy_city_only_my_topics(self):
         time.sleep(3)
         return CanonizerBrowsePage(self.driver)
 
+    def select_menu_item(self, menu_item):
+        self.click_browse_page_button()
+        self.hover(*BrowsePageIdentifiers.NAMESPACE)
+        select = Select(self.find_element(*BrowsePageIdentifiers.NAMESPACE))
+        select.select_by_visible_text(menu_item)
+        self.hover(*BrowsePageIdentifiers.TOPIC_NAME)
+        return CanonizerBrowsePage(self.driver)
 
+    def select_menu_item_with_only_my_topics(self, menu_item):
+        self.select_menu_item(menu_item)
+        self.hover(*BrowsePageIdentifiers.ONLY_MY_TOPICS)
+        self.find_element(*BrowsePageIdentifiers.ONLY_MY_TOPICS).click()
+        time.sleep(3)
+        return CanonizerBrowsePage(self.driver)
 
+    def select_menu_item_without_only_my_topics(self, menu_item):
+        self.select_menu_item(menu_item)
+        return CanonizerBrowsePage(self.driver)
 
+    def one_by_one(self, menu_item):
+        select = Select(self.find_element(*BrowsePageIdentifiers.NAMESPACE))
+        select.select_by_visible_text(menu_item)
+        time.sleep(4)
+        # self.hover(*BrowsePageIdentifiers.ONLY_MY_TOPICS)
+        # self.find_element(*BrowsePageIdentifiers.ONLY_MY_TOPICS).click()
+        # time.sleep(3)
+        return CanonizerBrowsePage(self.driver)
 
+    def one_by_one_only_my_topics(self):
+        self.hover(*BrowsePageIdentifiers.ONLY_MY_TOPICS)
+        self.find_element(*BrowsePageIdentifiers.ONLY_MY_TOPICS).click()
+        time.sleep(3)
+        return CanonizerBrowsePage(self.driver)
 
+    # def select_menu_items_one_by_one(self, topic_list):
+    #     self.click_browse_page_button()
+    #     self.hover(*BrowsePageIdentifiers.NAMESPACE)
+    #     for topic in topic_list:
+    #         result = self.one_by_one(topic)
+    #         print('result', result.get_url())
+    #     return CanonizerBrowsePage(self.driver)
 
+    def select_menu_items_one_by_one(self):
+        self.click_browse_page_button()
+        self.hover(*BrowsePageIdentifiers.NAMESPACE)
+        topics = Select(self.find_element(*BrowsePageIdentifiers.NAMESPACE))
+        options = topics.options
+        index = []
+        topic_list = []
+        for ele in options:
+            index.append(ele.get_attribute("value"))
+            topic_list.append(ele.text)
+        index.pop(0)
+        topic_list.pop(0)
+        for i in range(len(topic_list)):
+            result1 = self.one_by_one(topic_list[i])
+            name_space1 = "https://staging.canonizer.com/browse?namespace=" + str(index[i])
```

Comment:
Define it in config file. This we need to for production also. Hardcode is not a good idea